promises.tex

```diff
@@ -476,7 +476,7 @@ \section{Using Promises}\label{s:promises-usage}
 so we need to write our own function that returns an object with two keys
 (one for the filename, and one for the stats).
 As described in \chapref{s:pages},
-the notation \texttt{\{a,\ b\}} produces an object \texttt{\{"a":\ a,\ "b",\ b\}}:
+the notation \texttt{\{a,\ b\}} produces an object \texttt{\{"a":\ a,\ "b:",\ b\}}:
```

Comment:
I think the colon should be outside the double quotes, not inside, and there shouldn't be a comma after the last double quote.